packages/polyfills/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@waku/polyfills",
+  "version": "0.0.1",
+  "description": "Set of polyfills for Waku for React Native environment",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./metro": "./dist/metro.js"
+  },
+  "type": "module",
+  "homepage": "https://github.com/waku-org/js-waku/tree/master/packages/polyfills#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waku-org/js-waku.git"
+  },
+  "bugs": {
+    "url": "https://github.com/waku-org/js-waku/issues"
+  },
+  "license": "MIT OR Apache-2.0",
+  "keywords": [
+    "waku",
+    "decentralised",
+    "communication",
+    "web3",
+    "ethereum",
+    "dapps"
+  ],
+  "scripts": {
+    "build": "run-s build:**",
+    "build:esm": "tsc",
```

Comment:
important to note that `rollup` will not be bundling this package as bundling not needed and should be figured out by `react-native` application